### ISSUE TITLE
Timed messages setting doesn't apply to images shared from other apps

### ISSUE THREAD

**Issue by mattcoxonline** [wireapp/wire-android#1282]
## The problem

When you share an image from an app like Gallery to Wire, using the built-in OS sharing dialog, the setting for Timed Messages doesn't remain true for sending that image. It will send that message as a "standard" message, with no timer, and therefore reset the Timed Messages setting to none for future conversation.

## Environment

* Wire version that exhibits the issue: 3.3.374
* Last Wire version that did not exhibit the issue (if applicable):
* Android OS version used to run Wire: 7.0
* Samsung Experience version: 8.1
* Is this a custom firmware or a stock one: Stock
* Mobile phone model/manufacturer: Samsung Galaxy S8
* Mobile network type (EDGE/LTE/Wi-Fi/Offline): 4G + WiFi

## How to reproduce

1. Open existing conversation with Janet and set Timer to 1 day, for example
2. Switch to Gallery app and find a photo you want to send to Janet
3. Hit the native share dialog in the Gallery app and choose Wire
4. Pick Janet from the Wire share photo interface
5. Done.

You'll see now that the message has sent without the timer and also reset the timer setting.

## Expected behaviour

The expected behaviour is to share that photo with the pre-existing 1 Day timer you had already set for the conversation and retain that 1 Day timer for future messages.

**Comment by someoneEsle** [comment]
Hi, during the step

> 3. Hit the native share dialog in the Gallery app and choose Wire

you can actually set the timer by tapping the hourglass in the bottom right corner of the screen

**Comment by mattcoxonline** [comment]
> you can actually set the timer by tapping the hourglass in the bottom right corner of the screen

So it's intentional that it doesn't respect the exisitng choice for that conversation as default? 🤔

**Comment by someoneEsle** [comment]
Yes. The point is that you can send the same image to several conversations, and you may want to enforce the same timer for all of them -- therefore we give you the option to override the expiration time.
On top of that, we don't expect users to necessarily remember the expiration time they set in every conversation, so we would rather give them the option to set a timeout every time they share a picture through the sharing extension.

Although I do see your point, there is no obvious way to work this out in a way that doesn't make assumptions on the user's ability to remember expiration times. If you have any ideas on how to change the current behaviour to something that would be more intuitive for the majority of our users, we are open to suggestions (even though this does not have a very high priority at the moment). :)

**Comment by mattcoxonline** [comment]
Thanks for the responses @someoneEsle :)

**Comment by carlosdavidepto** [comment]
Since #1703 was marked as a duplicate I will leave my suggestions here instead.

The main problem I see with how this feature works isn't really that the external share get's its own timer. The main problem is that it will set those timer settings for the conversations where the shared item is sent.

>The point is that you can send the same image to several conversations, and you may want to enforce the same timer for all of them -- therefore we give you the option to override the expiration time.
>On top of that, we don't expect users to necessarily remember the expiration time they set in every conversation, so we would rather give them the option to set a timeout every time they share a picture through the sharing extension.

For users like me who use timers extensively, the way this feature works *requires* that I memorize my timer settings for all conversations which use them AND that I remember to restore them when jumping into a conversation the next time I do so.

TL;DR : Items shared from outside the app should have their own independent timer and not touch the conversation timers where the item is being sent, as this accomodates all use cases.

It would make way more sense for the shared item to have it's own timer but not change the rest of the following conversation. This would accomodate all use cases.
